src/Compilers/Core/Portable/PEWriter/SigningUtilities.cs

```diff
@@ -62,7 +62,7 @@ internal static int CalculateStrongNameSignatureSize(CommonPEModuleBuilder modul
 
             if (keySize == 0 && privateKey.HasValue)
             {
-                keySize = privateKey.Value.Modulus.Length;
+                keySize = privateKey.Value.Modulus?.Length ?? 0;
```

Comment:
Compiler folks: are we better to throw in this case? I'm not sure how this could happen but if it can, I worry this might change the key size calculation which might do something bad with signing somehow?

Reply:
I personally don't see a need to change the semantics. If we have always demanded a modulus in this code path, might as well just `!.` on it, or assert on it if you prefer.